assets/src/stories-editor/components/resizable-box/tests/getResizedWidthAndHeight.js

```diff
@@ -1,7 +1,7 @@
 /**
  * Internal dependencies
  */
-import { getResizedWidthAndHeight } from '../';
+import { getResizedWidthAndHeight } from '../helpers';
```

Comment:
Doesn't look like this test is running, at least not locally with `npm run test:js`.

Reply:
File ending needs to be `.test.js`

Reply:
Hmm, none of the `js` tests seem to have `.test.js` ending, only the `e2e` ones.

I just copied the `getBlockPositioning.js` file over from the general `helpers.js` tests, as you did with `getResizedWidthAndHeight.js` file, didn't change the ending.

Will check the config tomorrow.

Reply:
The test folder name should be `test`, not `tests`. Will fix.